components/blockquote.jsx

```diff
@@ -0,0 +1,24 @@
+var React = require('react');
+
+var Blockquote = React.createClass({
+  propTypes: {
+    className: React.PropTypes.string.isRequired,
+    imgSrc: React.PropTypes.string.isRequired,
+    imgAlt: React.PropTypes.string.isRequired,
+    quote: React.PropTypes.string.isRequired,
+    author: React.PropTypes.string.isRequired
+  },
+  render: function() {
+    return (
+      <blockquote className={this.props.className}>
+        <figure>
+          <img src={this.props.imgSrc} alt={this.props.imgAlt} width="148" />
+        </figure>
+        {this.props.quote}
```

Comment:
May I recommend actually changing this to `{this.props.children}`? That way the quote can actually contain rich markup like italics, bold and such, and would also read nicely when used, e.g.:

```html
<Blockquote imgSrc="/blah.jpg" author="Mmmmmmmmavis">
  React is <em>cooooooooooooool</em>!
</Blockquote>
```

Anyways is no big deal, we can file a bug for it and land it later if you like the idea.